src/System.Private.CoreLib/src/System/Threading/Thread.cs

```diff
@@ -423,7 +415,7 @@ public CultureInfo CurrentUICulture
                 // If you add more pre-conditions to this method, check to see if you also need to 
```

Comment:
These methods on Thread looks like unrechable code. I think you can delete them, in CoreLib and ecalllist.h at least.

I agree that the callbacks from VM to get the current culture look dysfunctional. Nobody noticed so far since we are not doing localization of VM error messages in .NET Core.

Reply:
They are referenced from the C code. Well, at least one of them was last time I checked... `Thread::GetCulture` method iirc.

Reply:
Ok, this can wait for later.